WorkflowInstance.variables query returns "Internal Server Error(s) while executing query"

ZeeQS 1.0.0-alpha5 docker image.
PostgreSQL 13.1

I want to get the variables of a completed workflow instance :

```graphql
query {
          workflowInstance (key: 2251799813916461) {
    				state
            variables {
              name
              value
            }
          }
        }
```

But I get :

```json
{
  "errors": [
    {
      "message": "Internal Server Error(s) while executing query"
    }
  ],
  "data": {
    "workflowInstance": {
      "state": "COMPLETED",
      "variables": null
    }
  }
}
```

Searching in the database, I do not find my variables values, there is only numbers in the value column of the variables table...

The relevant error seems to be:

`Caused by: org.postgresql.util.PSQLException: Large Objects may not be used in auto-commit mode.`

Here's the logs:

```
2020-12-25T21:11:40.802853054Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 2020-12-25 21:11:40.802 ERROR 1 --- [nio-9000-exec-8] g.k.e.error.DefaultGraphQLErrorHandler   : Error executing query Exception while fetching data (/workflowInstance/variables) : Unable to access lob stream; nested exception is org.hibernate.HibernateException: Unable to access lob stream
2020-12-25T21:11:40.802892963Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 
2020-12-25T21:11:40.802901479Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | org.springframework.orm.jpa.JpaSystemException: Unable to access lob stream; nested exception is org.hibernate.HibernateException: Unable to access lob stream
2020-12-25T21:11:40.802907937Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.orm.jpa.vendor.HibernateJpaDialect.convertHibernateAccessException(HibernateJpaDialect.java:353) ~[spring-orm-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.802914838Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.orm.jpa.vendor.HibernateJpaDialect.translateExceptionIfPossible(HibernateJpaDialect.java:255) ~[spring-orm-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.802921947Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.translateExceptionIfPossible(AbstractEntityManagerFactoryBean.java:528) ~[spring-orm-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.802928289Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.dao.support.ChainedPersistenceExceptionTranslator.translateExceptionIfPossible(ChainedPersistenceExceptionTranslator.java:61) ~[spring-tx-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.802934421Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.dao.support.DataAccessUtils.translateIfNecessary(DataAccessUtils.java:242) ~[spring-tx-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.802940029Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.dao.support.PersistenceExceptionTranslationInterceptor.invoke(PersistenceExceptionTranslationInterceptor.java:154) ~[spring-tx-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.802946041Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.802966787Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.data.jpa.repository.support.CrudMethodMetadataPostProcessor$CrudMethodMetadataPopulatingMethodInterceptor.invoke(CrudMethodMetadataPostProcessor.java:149) ~[spring-data-jpa-2.3.4.RELEASE.jar:2.3.4.RELEASE]
2020-12-25T21:11:40.802974327Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.802982501Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:95) ~[spring-aop-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.802989247Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.802996093Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.data.repository.core.support.MethodInvocationValidator.invoke(MethodInvocationValidator.java:99) ~[spring-data-commons-2.3.4.RELEASE.jar:2.3.4.RELEASE]
2020-12-25T21:11:40.803002343Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803008835Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:212) ~[spring-aop-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803014905Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at com.sun.proxy.$Proxy131.findByWorkflowInstanceKey(Unknown Source) ~[na:na]
2020-12-25T21:11:40.803021230Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
2020-12-25T21:11:40.803027533Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:na]
2020-12-25T21:11:40.803033486Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
2020-12-25T21:11:40.803041427Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at java.base/java.lang.reflect.Method.invoke(Method.java:566) ~[na:na]
2020-12-25T21:11:40.803047789Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:344) ~[spring-aop-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803054658Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:205) ~[spring-aop-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803061438Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at com.sun.proxy.$Proxy78.findByWorkflowInstanceKey(Unknown Source) ~[na:na]
2020-12-25T21:11:40.803067675Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at io.zeebe.zeeqs.data.resolvers.WorkflowInstanceResolver.variables(WorkflowInstanceResolver.kt:31) ~[graphql-api-1.0.0-SNAPSHOT.jar:1.0.0-SNAPSHOT]
2020-12-25T21:11:40.803074492Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at io.zeebe.zeeqs.data.resolvers.WorkflowInstanceResolverMethodAccess.invoke(Unknown Source) ~[reflectasm-1.11.9.jar:1.0.0-SNAPSHOT]
2020-12-25T21:11:40.803081081Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.kickstart.tools.MethodFieldResolverDataFetcher.get(MethodFieldResolver.kt:208) ~[graphql-java-tools-6.0.2.jar:na]
2020-12-25T21:11:40.803088360Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.execution.ExecutionStrategy.fetchField(ExecutionStrategy.java:272) ~[graphql-java-14.0.jar:na]
2020-12-25T21:11:40.803095224Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.execution.ExecutionStrategy.resolveFieldWithInfo(ExecutionStrategy.java:200) ~[graphql-java-14.0.jar:na]
2020-12-25T21:11:40.803107408Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.execution.AsyncExecutionStrategy.execute(AsyncExecutionStrategy.java:74) ~[graphql-java-14.0.jar:na]
2020-12-25T21:11:40.803114288Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.execution.ExecutionStrategy.completeValueForObject(ExecutionStrategy.java:656) ~[graphql-java-14.0.jar:na]
2020-12-25T21:11:40.803120812Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.execution.ExecutionStrategy.completeValue(ExecutionStrategy.java:440) ~[graphql-java-14.0.jar:na]
2020-12-25T21:11:40.803127229Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.execution.ExecutionStrategy.completeField(ExecutionStrategy.java:390) ~[graphql-java-14.0.jar:na]
2020-12-25T21:11:40.803133769Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.execution.ExecutionStrategy.lambda$resolveFieldWithInfo$0(ExecutionStrategy.java:202) ~[graphql-java-14.0.jar:na]
2020-12-25T21:11:40.803140183Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at java.base/java.util.concurrent.CompletableFuture.uniApplyNow(CompletableFuture.java:680) ~[na:na]
2020-12-25T21:11:40.803146719Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at java.base/java.util.concurrent.CompletableFuture.uniApplyStage(CompletableFuture.java:658) ~[na:na]
2020-12-25T21:11:40.803153115Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at java.base/java.util.concurrent.CompletableFuture.thenApply(CompletableFuture.java:2094) ~[na:na]
2020-12-25T21:11:40.803159466Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.execution.ExecutionStrategy.resolveFieldWithInfo(ExecutionStrategy.java:201) ~[graphql-java-14.0.jar:na]
2020-12-25T21:11:40.803165957Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.execution.AsyncExecutionStrategy.execute(AsyncExecutionStrategy.java:74) ~[graphql-java-14.0.jar:na]
2020-12-25T21:11:40.803172413Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.execution.Execution.executeOperation(Execution.java:165) ~[graphql-java-14.0.jar:na]
2020-12-25T21:11:40.803178812Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.execution.Execution.execute(Execution.java:106) ~[graphql-java-14.0.jar:na]
2020-12-25T21:11:40.803185147Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.GraphQL.execute(GraphQL.java:623) ~[graphql-java-14.0.jar:na]
2020-12-25T21:11:40.803191524Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.GraphQL.parseValidateAndExecute(GraphQL.java:556) ~[graphql-java-14.0.jar:na]
2020-12-25T21:11:40.803197703Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.GraphQL.executeAsync(GraphQL.java:520) ~[graphql-java-14.0.jar:na]
2020-12-25T21:11:40.803204074Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.kickstart.execution.GraphQLInvoker.executeAsync(GraphQLInvoker.java:26) ~[graphql-java-kickstart-9.1.0.jar:na]
2020-12-25T21:11:40.803210318Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.kickstart.execution.GraphQLInvoker.query(GraphQLInvoker.java:38) ~[graphql-java-kickstart-9.1.0.jar:na]
2020-12-25T21:11:40.803217411Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.kickstart.execution.GraphQLInvoker.query(GraphQLInvoker.java:31) ~[graphql-java-kickstart-9.1.0.jar:na]
2020-12-25T21:11:40.803223875Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.kickstart.servlet.HttpRequestHandlerImpl.invoke(HttpRequestHandlerImpl.java:67) ~[graphql-java-servlet-9.1.0.jar:na]
2020-12-25T21:11:40.803230129Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.kickstart.servlet.HttpRequestHandlerImpl.execute(HttpRequestHandlerImpl.java:52) ~[graphql-java-servlet-9.1.0.jar:na]
2020-12-25T21:11:40.803236548Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.kickstart.servlet.HttpRequestHandlerImpl.handle(HttpRequestHandlerImpl.java:37) ~[graphql-java-servlet-9.1.0.jar:na]
2020-12-25T21:11:40.803243038Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.kickstart.servlet.AbstractGraphQLHttpServlet.doRequest(AbstractGraphQLHttpServlet.java:147) ~[graphql-java-servlet-9.1.0.jar:na]
2020-12-25T21:11:40.803254738Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.kickstart.servlet.AbstractGraphQLHttpServlet.doRequestAsync(AbstractGraphQLHttpServlet.java:137) ~[graphql-java-servlet-9.1.0.jar:na]
2020-12-25T21:11:40.803261888Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at graphql.kickstart.servlet.AbstractGraphQLHttpServlet.doPost(AbstractGraphQLHttpServlet.java:169) ~[graphql-java-servlet-9.1.0.jar:na]
2020-12-25T21:11:40.803268112Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652) ~[tomcat-embed-core-9.0.38.jar:4.0.FR]
2020-12-25T21:11:40.803274864Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[tomcat-embed-core-9.0.38.jar:4.0.FR]
2020-12-25T21:11:40.803281414Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803288076Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803294591Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53) ~[tomcat-embed-websocket-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803301306Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803308153Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803314638Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.web.filter.CorsFilter.doFilterInternal(CorsFilter.java:92) ~[spring-web-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803321351Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119) ~[spring-web-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803328748Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803335584Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803342104Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100) ~[spring-web-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803348588Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119) ~[spring-web-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803355707Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803362151Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803368690Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93) ~[spring-web-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803375288Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119) ~[spring-web-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803381866Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803393226Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803400312Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.boot.actuate.metrics.web.servlet.WebMvcMetricsFilter.doFilterInternal(WebMvcMetricsFilter.java:93) ~[spring-boot-actuator-2.3.4.RELEASE.jar:2.3.4.RELEASE]
2020-12-25T21:11:40.803407504Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119) ~[spring-web-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803413861Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803420646Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803427621Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201) ~[spring-web-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803434462Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119) ~[spring-web-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803441446Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803448227Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803454604Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803461135Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803467865Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803474354Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803480932Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803487237Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803493607Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803499967Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803506673Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803513304Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803519910Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1590) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803531899Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803539720Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
2020-12-25T21:11:40.803546369Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
2020-12-25T21:11:40.803552772Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) ~[tomcat-embed-core-9.0.38.jar:9.0.38]
2020-12-25T21:11:40.803559262Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]
2020-12-25T21:11:40.803565564Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | Caused by: org.hibernate.HibernateException: Unable to access lob stream
2020-12-25T21:11:40.803571476Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.type.descriptor.java.DataHelper.extractString(DataHelper.java:270) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803578523Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.type.descriptor.java.StringTypeDescriptor.wrap(StringTypeDescriptor.java:75) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803585383Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.type.descriptor.java.StringTypeDescriptor.wrap(StringTypeDescriptor.java:22) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803591740Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.type.descriptor.sql.ClobTypeDescriptor$1.doExtract(ClobTypeDescriptor.java:44) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803598297Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.type.descriptor.sql.BasicExtractor.extract(BasicExtractor.java:47) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803605207Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.type.AbstractStandardBasicType.nullSafeGet(AbstractStandardBasicType.java:257) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803611626Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.type.AbstractStandardBasicType.nullSafeGet(AbstractStandardBasicType.java:253) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803618132Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.type.AbstractStandardBasicType.nullSafeGet(AbstractStandardBasicType.java:243) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803624994Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.type.AbstractStandardBasicType.hydrate(AbstractStandardBasicType.java:329) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803631551Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.persister.entity.AbstractEntityPersister.hydrate(AbstractEntityPersister.java:3088) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803638294Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.loader.Loader.loadFromResultSet(Loader.java:1907) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803645057Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.loader.Loader.hydrateEntityState(Loader.java:1835) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803652480Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.loader.Loader.instanceNotYetLoaded(Loader.java:1808) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803659212Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.loader.Loader.getRow(Loader.java:1660) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803670617Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.loader.Loader.getRowFromResultSet(Loader.java:745) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803677450Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.loader.Loader.getRowsFromResultSet(Loader.java:1044) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803683886Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.loader.Loader.processResultSet(Loader.java:995) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803690221Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.loader.Loader.doQuery(Loader.java:964) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803696388Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:350) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803702830Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.loader.Loader.doList(Loader.java:2887) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803709180Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.loader.Loader.doList(Loader.java:2869) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803715434Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2701) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803721772Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.loader.Loader.list(Loader.java:2696) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803727961Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.loader.hql.QueryLoader.list(QueryLoader.java:506) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803734001Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.list(QueryTranslatorImpl.java:400) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803740317Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.engine.query.spi.HQLQueryPlan.performList(HQLQueryPlan.java:219) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803747010Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.internal.SessionImpl.list(SessionImpl.java:1415) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803753316Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.query.internal.AbstractProducedQuery.doList(AbstractProducedQuery.java:1565) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803759964Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.query.internal.AbstractProducedQuery.list(AbstractProducedQuery.java:1533) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803766606Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.query.Query.getResultList(Query.java:165) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803773238Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.query.criteria.internal.compile.CriteriaQueryTypeQueryAdapter.getResultList(CriteriaQueryTypeQueryAdapter.java:76) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803780060Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
2020-12-25T21:11:40.803786139Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:na]
2020-12-25T21:11:40.803792271Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
2020-12-25T21:11:40.803798571Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at java.base/java.lang.reflect.Method.invoke(Method.java:566) ~[na:na]
2020-12-25T21:11:40.803809917Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.orm.jpa.SharedEntityManagerCreator$DeferredQueryInvocationHandler.invoke(SharedEntityManagerCreator.java:409) ~[spring-orm-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803817295Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at com.sun.proxy.$Proxy153.getResultList(Unknown Source) ~[na:na]
2020-12-25T21:11:40.803823559Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.data.jpa.repository.query.JpaQueryExecution$CollectionExecution.doExecute(JpaQueryExecution.java:126) ~[spring-data-jpa-2.3.4.RELEASE.jar:2.3.4.RELEASE]
2020-12-25T21:11:40.803830433Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.data.jpa.repository.query.JpaQueryExecution.execute(JpaQueryExecution.java:88) ~[spring-data-jpa-2.3.4.RELEASE.jar:2.3.4.RELEASE]
2020-12-25T21:11:40.803836974Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.data.jpa.repository.query.AbstractJpaQuery.doExecute(AbstractJpaQuery.java:154) ~[spring-data-jpa-2.3.4.RELEASE.jar:2.3.4.RELEASE]
2020-12-25T21:11:40.803843106Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.data.jpa.repository.query.AbstractJpaQuery.execute(AbstractJpaQuery.java:142) ~[spring-data-jpa-2.3.4.RELEASE.jar:2.3.4.RELEASE]
2020-12-25T21:11:40.803849941Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.data.repository.core.support.QueryExecutorMethodInterceptor$QueryMethodInvoker.invoke(QueryExecutorMethodInterceptor.java:195) ~[spring-data-commons-2.3.4.RELEASE.jar:2.3.4.RELEASE]
2020-12-25T21:11:40.803857252Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.data.repository.core.support.QueryExecutorMethodInterceptor.doInvoke(QueryExecutorMethodInterceptor.java:152) ~[spring-data-commons-2.3.4.RELEASE.jar:2.3.4.RELEASE]
2020-12-25T21:11:40.803864654Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.data.repository.core.support.QueryExecutorMethodInterceptor.invoke(QueryExecutorMethodInterceptor.java:130) ~[spring-data-commons-2.3.4.RELEASE.jar:2.3.4.RELEASE]
2020-12-25T21:11:40.803871490Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803878429Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:367) ~[spring-tx-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803885128Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:118) ~[spring-tx-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803891853Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803898649Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.springframework.dao.support.PersistenceExceptionTranslationInterceptor.invoke(PersistenceExceptionTranslationInterceptor.java:139) ~[spring-tx-5.2.9.RELEASE.jar:5.2.9.RELEASE]
2020-12-25T21:11:40.803905331Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	... 88 common frames omitted
2020-12-25T21:11:40.803911100Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | Caused by: org.postgresql.util.PSQLException: Large Objects may not be used in auto-commit mode.
2020-12-25T21:11:40.803917349Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.postgresql.largeobject.LargeObjectManager.open(LargeObjectManager.java:243) ~[postgresql-42.2.18.jar:42.2.18]
2020-12-25T21:11:40.803924202Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.postgresql.largeobject.LargeObjectManager.open(LargeObjectManager.java:229) ~[postgresql-42.2.18.jar:42.2.18]
2020-12-25T21:11:40.803930474Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.postgresql.jdbc.AbstractBlobClob.getLo(AbstractBlobClob.java:272) ~[postgresql-42.2.18.jar:42.2.18]
2020-12-25T21:11:40.803941686Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.postgresql.jdbc.AbstractBlobClob.getBinaryStream(AbstractBlobClob.java:117) ~[postgresql-42.2.18.jar:42.2.18]
2020-12-25T21:11:40.803948298Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.postgresql.jdbc.PgClob.getCharacterStream(PgClob.java:54) ~[postgresql-42.2.18.jar:42.2.18]
2020-12-25T21:11:40.803954734Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	at org.hibernate.type.descriptor.java.DataHelper.extractString(DataHelper.java:263) ~[hibernate-core-5.4.21.Final.jar:5.4.21.Final]
2020-12-25T21:11:40.803961384Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 	... 136 common frames omitted
2020-12-25T21:11:40.803967103Z madam_zeeqs.1.yn8tdvgl2r9l@z68-gen3    | 

```

--- a/data/src/main/kotlin/io/zeebe/zeeqs/data/repository/IncidentRepository.kt
+++ b/data/src/main/kotlin/io/zeebe/zeeqs/data/repository/IncidentRepository.kt
@@ -5,17 +5,22 @@ import io.zeebe.zeeqs.data.entity.IncidentState
 import org.springframework.data.domain.Pageable
 import org.springframework.data.repository.PagingAndSortingRepository
 import org.springframework.stereotype.Repository
+import org.springframework.transaction.annotation.Transactional
 
 @Repository
 interface IncidentRepository : PagingAndSortingRepository<Incident, Long> {
 
+    @Transactional(readOnly = true)
     fun findByWorkflowInstanceKey(workflowInstanceKey: Long): List<Incident>
 
+    @Transactional(readOnly = true)
     fun findByJobKey(jobKey: Long): List<Incident>
 
+    @Transactional(readOnly = true)
     fun findByStateIn(stateIn: List<IncidentState>, pageable: Pageable): List<Incident>
 
     fun countByStateIn(stateIn: List<IncidentState>): Long
 
+    @Transactional(readOnly = true)
     fun findByWorkflowInstanceKeyAndStateIn(workflowInstanceKey: Long, stateIn: List<IncidentState>): List<Incident>
 }

--- a/data/src/main/kotlin/io/zeebe/zeeqs/data/repository/VariableRepository.kt
+++ b/data/src/main/kotlin/io/zeebe/zeeqs/data/repository/VariableRepository.kt
@@ -3,9 +3,11 @@ package io.zeebe.zeeqs.data.repository
 import io.zeebe.zeeqs.data.entity.Variable
 import org.springframework.data.repository.PagingAndSortingRepository
 import org.springframework.stereotype.Repository
+import org.springframework.transaction.annotation.Transactional
 
 @Repository
 interface VariableRepository : PagingAndSortingRepository<Variable, Long> {
 
+    @Transactional(readOnly = true)
     fun findByWorkflowInstanceKey(workflowInstanceKey: Long): List<Variable>
 }

--- a/data/src/main/kotlin/io/zeebe/zeeqs/data/repository/VariableUpdateRepository.kt
+++ b/data/src/main/kotlin/io/zeebe/zeeqs/data/repository/VariableUpdateRepository.kt
@@ -3,11 +3,14 @@ package io.zeebe.zeeqs.data.repository
 import io.zeebe.zeeqs.data.entity.VariableUpdate
 import org.springframework.data.repository.PagingAndSortingRepository
 import org.springframework.stereotype.Repository
+import org.springframework.transaction.annotation.Transactional
 
 @Repository
 interface VariableUpdateRepository : PagingAndSortingRepository<VariableUpdate, Long> {
 
+    @Transactional(readOnly = true)
     fun findByWorkflowInstanceKey(workflowInstanceKey: Long): List<VariableUpdate>
 
+    @Transactional(readOnly = true)
     fun findByVariableKey(variableKey: Long): List<VariableUpdate>
 }

